coala-bears.cabal

```diff
@@ -0,0 +1,13 @@
+Name: coala-bears
+Version: 0.10
+cabal-version: >= 1.2
+build-type: Simple
+
+-- safe 0.3.9 is needed to coexist with Cabal 1.16
+-- safe 0.3.10 requires Cabal 1.18
+
+-- Note trusty apt package "haskell-platform" includes mtl-2.1.2
```

Comment:
Confirmed.  http://packages.ubuntu.com/trusty/haskell-platform actually has two version
```
libghc-mtl-dev (<< 2.1.2+)
    Haskell monad transformer library for GHC 

libghc-mtl-dev (>= 2.1.2)
```

Im not sure how to pick the desired one on travis.

xenial + only have the desired version 2.2.1. http://packages.ubuntu.com/xenial/libghc-mtl-dev